src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20180625_autopause_threshold.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="azmina" id="autopause_threshold">
+        <addColumn tableName="CLUSTER">
+            <column name="autopauseThreshold" type="INT">
+                <constraints nullable="false" />
```

Comment:
Hm, since this is a new not-null column we should probably include a migration to update all existing clusters in the DB. Something like:

1. Add the new column as nullable
2. Update all existing clusters to the current default
3. Make the new column not-null